web_hierarchy/models/ir_ui_view.py

```diff
@@ -0,0 +1,10 @@
+# -*- coding: utf-8 -*-
+#   © 2019 Kevin Kamau
+#   License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+from odoo import fields, models, _
+
+
+class IrUiView(models.Model):
+    _inherit = 'ir.ui.view'
+
+    type = fields.Selection(selection_add=[('hierarchy', _('Hierarchy'))])
```

Comment:
There is no need for explicitly translating the selection labels, this is being done on `fields.Selection._description_selection`, you could remove this from all the places you are adding the type.